recipes/jupyter-server-fileid/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "jupyter_server_fileid" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a12209bdef4f2f9d57051b7556a089299fb9f26b501f643946854220c955be14
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - hatchling >=1.0
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - jupyter_server >=1.15,<3
```

Comment:
I'm still confused, don't we need `jupyter_server >=2`?

Reply:
I used the constrain of the published package.

Reply:
File ID does not have a hard dependency on JS2, though it is currently needed to track in-band edits.